tests/Test/Pager/Subscriber/Filtration/Doctrine/ORM/QueryTest.php

```diff
@@ -68,55 +68,81 @@ public function shouldFilterSimpleDoctrineQuery(): void
         $dispatcher = new EventDispatcher();
         $dispatcher->addSubscriber(new PaginationSubscriber());
         $dispatcher->addSubscriber(new Filtration());
-        $p = new Paginator($dispatcher);
+        $requestStack = $this->getRequestStack(['filterParam' => 'a.title', 'filterValue' => '*er']);
+        $p = new Paginator($dispatcher, $requestStack);
 
-        $_GET['filterParam'] = 'a.title';
-        $_GET['filterValue'] = '*er';
         $this->startQueryLog();
         $query = $this->em->createQuery('SELECT a FROM Test\Fixture\Entity\Article a');
         $query->setHint(QuerySubscriber::HINT_FETCH_JOIN_COLLECTION, false);
         $view = $p->paginate($query, 1, 10);
 
         $items = $view->getItems();
+
         $this->assertCount(2, $items);
         $this->assertEquals('summer', $items[0]->getTitle());
         $this->assertEquals('winter', $items[1]->getTitle());
 
-        $_GET['filterValue'] = 'summer';
-        $view = $p->paginate($query, 1, 10);
+        $this->assertEquals(2, $this->queryAnalyzer->getNumExecutedQueries());
+        $executed = $this->queryAnalyzer->getExecutedQueries();
+
+        // Different aliases separators according to Doctrine version
+        if (version_compare(\Doctrine\ORM\Version::VERSION, '2.5', '<')) {
+            $this->assertEquals('SELECT a0_.id AS id0, a0_.title AS title1, a0_.enabled AS enabled2 FROM Article a0_ WHERE a0_.title LIKE \'%er\' LIMIT 10', $executed[1]);
+        } else {
+            $this->assertEquals('SELECT a0_.id AS id_0, a0_.title AS title_1, a0_.enabled AS enabled_2 FROM Article a0_ WHERE a0_.title LIKE \'%er\' LIMIT 10', $executed[1]);
+        }
+    }
+
+    /**
+     * @test
+     */
+    public function shouldFilterSimpleDoctrineQueryWithoutWildcard(): void
```

Comment:
Is this a new test?

Reply:
I had to split some tests, when `$_GET` were overridden